pkg/tfvars/tfvars.go

```diff
@@ -108,3 +92,12 @@ func TFVars(clusterID string, cfg *types.InstallConfig, osImage, bootstrapIgn, m
 
 	return json.MarshalIndent(config, "", "  ")
 }
+
+func workerPool(pools []types.MachinePool) *types.MachinePool {
```

Comment:
we have 2 defintions of this function. is a common helper more appropiate? find name in compute list

Reply:
> we have 2 defintions of this function. is a common helper more appropiate?

I think we should drop the function entirely, and just iterate through compute pools when generating MachineSets.

Reply:
We will eventually drop this function very soon as part of subsequent steps of this change from master and worker to control plane and compute.

Reply:
The `workerPool` function has been dropped from `pkg/tfvars/tfvars.go` as it is no longer used.